library/src/engine.cxx

```diff
@@ -41,16 +41,16 @@ engine::engine(
 
   // build default cache path
 #if defined(_WIN32)
-  std::string cachePath = vtksys::SystemTools::GetEnv("LOCALAPPDATA");
-  cachePath = cachePath + "/f3d";
+  fs::path cachePath(std::getenv("LOCALAPPDATA"));
```

Comment:
Creating a `fs::path` with nullptr will crash the app. You need to check the result of `std::getenv`

Reply:
good catch!